docs/sdk/api-reference.md

```diff
@@ -3381,40 +2648,26 @@ Creates Geo proto for geographical searches.
 **Returns:**
 - Geo proto message
 
-
-### Search._list_topk_generator()
+### Search._list_topk_generator
 
 ```python
-def _list_topk_generator(self,
-                        endpoint: Callable[..., Any],
-                        proto_message: Any,
-                        request_data: Dict[str, Any]) -> Generator[Dict[str, Any], None, None]
+Search._list_topk_generator(endpoint, proto_message, request_data)
 ```
 
 Internal generator for top-k search results.
 
-### Search._list_all_pages_generator()
+### Search._list_all_pages_generator
```

Comment:
I think this is a internal method. WDYT @sanjaychelliah ?

Reply:
Yes all the comments related to search are all internal methods, we should remove them

Reply:
As this is an internal method i have removed this from the doc